presto-docs/src/main/sphinx/installation/deploy-brew.rst

```diff
@@ -0,0 +1,94 @@
+============================================
+Deploy Presto on an Intel Mac using Homebrew
+============================================
+
+*Note*: These steps were developed and tested on Mac OS X on Intel. These steps will not work with Apple Silicon CPUs.
```

Comment:
Correct me if i am wrong. AFAK, some patches have been merged to make Presto run on macOS with aarch64 cpus. Is there any gap?

Reply:
@7c00, great question! These steps work for Intel chip Macs and do not work on M1 Macs. 

I tested yesterday with the current downloadable files. When I ran the command "brew install prestodb", the following error was returned:
"prestodb: The x86_64 architecture is required for this software.
Error: prestodb: An unsatisfied requirement failed this build."

There is a way to install the current prestodb involving rosetta that I just worked through yesterday with @yhwang (thanks!) involving 
a) installing brew to /usr/local instead of the default /opt/homebrew for arm64
b) export PATH=/usr/local/bin:$PATH so the shell will find x86_64 brew first
c) run the commands prefaced with "arch -x86_64". Examples "arch -x86_64 brew install prestodb", "arch -x86_64 presto-server start", "arch -x86_64 ./presto"

I plan to document this process in the near future - probably as an addition to this page as a separate and alternate set of steps. I don't want to delay the publication of this page, though, as the existing doc should help new Presto users who have the relevant (Intel Mac) hardware. 

I hope that makes sense.

Reply:
`prestodb: The x86_64 architecture is required for this software.` The error message might come from brew install script (https://github.com/Homebrew/homebrew-core/blob/master/Formula/prestodb.rb#L26). When we have confirmed that the aarch64-build could actually run on macOS, we shall submit a PR to fix the brew script. These tasks could be finished in future.